src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+
+      <fieldset class="form-fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input 
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input 
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        
+        <label class="form-field">
+          How old are You?
+          <input 
+            type="number"
+            name="age"
+            min="1" 
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input 
+            type="checkbox"
+            name="acceptation"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email"
+            name="e-mail" 
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input 
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="32"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <input type="radio" name="cats"> Yes
+          <input type="radio" name="cats"> No
+        </label>
```

Comment:
Move this text out of label, label is connected to radio option, and add different labels for inputs, also you should add `value` attributes for radios
```suggestion
       <div class="form-field">
        Do you love cats?
        <label>
          <input type="radio" name="cats" value="...">
          Yes
        </label>
        <label>
          <input type="radio" name="cats" value="..."> 
          No
        </label>
      </div>
```